app/controllers/application_controller.rb

```diff
@@ -4,6 +4,8 @@ class ApplicationController < ActionController::Base
   rescue_from Exception do |ex|
     Rollbar.error(ex)
     Rails.logger.fatal(ex)
+    raise unless Rails.env.production?
```

Comment:
This is helpful, however I think it's best to not condition code per environment and a better approach would be to rethink and rewrite this as we also have a problem with not being able to see the full trace of errors when they take place in production which is a much bigger issue than other environments.

Reply:
That's interesting. I guessed that in production you could see the full stack error in Rollbar log. I am wrong in this guess?

Reply:
I need to look this up more as it's been a while since coming to the conclusion that we should change this.

You can indeed see what's going on in rollbar, however as this is an open source project and we want anyone to be able to work on the project, we want better visibility of ongoing issues. We have rollbar configured to post on an public Slack channel but the details rendered there are not sufficient .

Eg screenshots below
<img width="629" alt="Screen Shot 2020-05-25 at 19 12 45" src="https://user-images.githubusercontent.com/159200/82835345-aa369c00-9ebb-11ea-9870-1a0f63e721b3.png">
<img width="636" alt="Screen Shot 2020-05-25 at 19 12 38" src="https://user-images.githubusercontent.com/159200/82835346-ab67c900-9ebb-11ea-906d-eb13f1bacf46.png">

Reply:
So some sort of ENV variable that you could set with Heroku so you can see a full crash on staging/production server instead of 404/500 error page? 🤔